packages/pluggableWidgets/column-chart-native/src/components/ColumnChart.tsx

```diff
@@ -0,0 +1,367 @@
+                                        // TODO: this needs to be checked for bar chart
+                                        height={chartDimensions?.height || undefined}
+                                        width={chartDimensions?.width || undefined}
+                                        padding={aggregateGridPadding(style.grid)}
+                                        scale={
+                                            dataTypesResult
+                                                ? { x: getScale(dataTypesResult.x), y: getScale(dataTypesResult.y) }
+                                                : undefined
+                                        }
+                                        style={mapToGridStyle(style.grid)}
+                                    >
+                                        <VictoryAxis
+                                            orientation={"bottom"}
+                                            style={mapToAxisStyle(style.grid, style.xAxis)}
+                                            {...(firstSeries?.xFormatter
+                                                ? { tickFormat: firstSeries.xFormatter }
+                                                : undefined)}
+                                        />
+                                        <VictoryAxis
+                                            style={mapToAxisStyle(style.grid, style.yAxis)}
+                                            orientation={"left"}
+                                            dependentAxis
+                                            {...(firstSeries?.yFormatter
+                                                ? { tickFormat: firstSeries.yFormatter }
+                                                : undefined)}
+                                        />
+                                        {groupedOrStacked}
+                                    </VictoryChart>
+                                ) : null}
+                            </View>
+                            {axisLabelStyles.extractedXAxisLabelStyle.relativePositionGrid === "right"
+                                ? xAxisLabelComponent
+                                : null}
+                        </View>
+                        {axisLabelStyles.extractedXAxisLabelStyle.relativePositionGrid === "bottom"
+                            ? xAxisLabelComponent
+                            : null}
+                    </View>
+                    {showLegend ? (
+                        <Legend style={style.legend} series={sortedSeries} seriesColors={normalizedColumnColors} />
+                    ) : null}
+                </View>
+            )}
+        </View>
+    );
+}
+
+type DataTypeResult = { x: string; y: string };
+
+function getDataTypes(series: ColumnChartSeries[]): DataTypeResult | Error | undefined {
+    let dataTypes: DataTypeResult | undefined;
+
+    for (const _series of series) {
+        const { dataPoints } = _series;
```

Comment:
Let's avoid the `_` by calling this variable `seriesItem` like everywhere else? :)

Reply:
done